### PR TITLE
OS rename

### PR DESCRIPTION
**Pull request by jburel**
Rename OS so we can add support for newer version of Ubuntu or Debian

To test: check that travis remains green

### CODE REVIEW

**Comment by joshmoore** [comment]
@jburel : have you generated the related doc PR associated with this anywhere?

File: README.md [hunk 1]
```diff
@@ -102,18 +100,18 @@ To run installation scripts on a remote host:
 Testing in DOCKER
 -----------------
 
-These tests are only for CentOS 7 and Ubuntu deployment. Unfortunately there is no docker container for Mac OS X installation scritps
+These tests are only for CentOS 7, Ubuntu, Debian deployment. Unfortunately there is no docker container for Mac OS X installation scritps
```
Comment (joshmoore) [inline README.md]:
```suggestion
These tests are only for CentOS 7, Ubuntu, and Debian deployments. Unfortunately there is no docker container for Mac OS X installation scripts
```

**Comment by jburel** [comment]
I have not generated a tmp doc PR. I can open one this evening

**Comment by jburel** [comment]
Doc PR https://github.com/openmicroscopy/ome-documentation/pull/1923

**Comment by rgozim** [comment]
Docs are ok and build is green on travis.

**Pull request merged**